src/components/SingleDatePicker.jsx

```diff
@@ -513,7 +466,7 @@ class SingleDatePicker extends BaseClass {
             {...css(styles.SingleDatePicker_closeButton)}
             aria-label={phrases.closeDatePicker}
             type="button"
-            onClick={this.onClearFocus}
+            onClick={this.onOutsideClick}
```

Comment:
do you mean to use `onOutsideClick` here?

Reply:
Yup! I changed the naming of the `onClearFocus` method to `onOutsideClick` so it would match the nomenclature of the `DateRangePicker` component, but the method itself does the same thing.

Reply:
I meant the variable instead of the instance method

Reply:
Ah, no I def meant the instance method in both places